core/templates/dev/head/css/oppia.css

```diff
@@ -1195,11 +1195,34 @@ pre.oppia-pre-wrapped-text {
   border-radius: 4px;
 }
 
+.oppia-editable-section:hover pre {
+	background-color: inherit;
+}
+
+.oppia-editable-section:hover oppia-noninteractive-link,
+.oppia-editable-section:hover oppia-noninteractive-video,
+.oppia-editable-section:hover oppia-noninteractive-tabs {
+	pointer-events: none;
+	cursor: pointer;
+}
+
+.oppia-editable-section:hover oppia-noninteractive-tabs li.active a {
+	background-color: inherit;
+	-webkit-transition : all 200ms;
+	transition: all 200ms;
+}
+
 .oppia-editable-section:hover .oppia-interaction-preview {
   background: rgba(5, 140, 166, 0.5);
   border-radius: 4px;
 }
 
+.oppia-editable-section:hover oppia-noninteractive-image img {
+    opacity: 0.7;
```

Comment:
Ah, maybe my mental model of what's going on is wrong, then. Here's what I'm thinking:

- We have a bunch of content in a div. This includes images, etc.
- We also have a separate div that's essentially a gray overlay mask on the content. It's a sibling to the previous div, and it's absolutely positioned (with a higher z-index). (Sort of like [this](http://stackoverflow.com/questions/3182683/css-image-mask-overlay).)

Since the separate div has a higher z-index, it automatically masks all the content in the first div, and no (or minimal) tweaks are needed to the first div. In particular, it seems a bit odd to adjust the images so that they 'mask' themselves -- the separate overlay should do it for them.

Having said all that, I just looked at state_editor_content.html, and realized that my assumption (about sibling divs and absolute positioning) might not be correct. I think the correct fix would be to reorganize things so that it becomes correct, since then there'll be less individual manipulation needed for the stuff under the overlay mask. What do you think?